svc-bgs-api/src/main_consumer.rb

```diff
@@ -6,7 +6,16 @@
 require 'rabbit_subscriber'
 require 'bgs_client'
 
-$logger = Logger.new(STDOUT)
+if ENV.fetch('DOCKER_LOGS','') == 'true'
+  fd = IO.sysopen("/proc/1/fd/1","w")
```

Comment:
Add code comments to explain what's happening here and why. I'm not familiar with `/proc/1/fd/1`.

Reply:
Addressed in line 5 of `setup.rb`. This logger setup is removed from `main_consumer.rb`